binder/src/main/java/io/grpc/binder/internal/BinderTransport.java

```diff
@@ -345,6 +345,7 @@ private final void sendShutdownTransaction() {
         // Ignore.
       }
       Parcel parcel = Parcel.obtain();
+      parcel.writeInt(0);  // Send empty flags to avoid a memory leak linked to empty parcels.
```

Comment:
Add b/nnn link to the android bug?

Reply:
Sure! I wasn't sure whether internal-only references were appropriate here.